tools/v3dconverter/readme.txt

```diff
@@ -0,0 +1,40 @@
+ 
```

Comment:
Can we put this on a separate PR? Seems out the "human improvement" scope

Reply:
Lost in git, **ONE MORE TIME**...
I don't understand why it is in humans enhancement PR, I pushed this 2 days ago in the v3dconverter branch, but I didn't asked any PR for this branch because even if v3dconverter works (I did all Corsica scenery update new models with it), as written in readme.txt/FOREWORD: it's not yet finished !
How can I remove these files ? With .../Delete file ???

Reply:
It seems you pushed onto this branch too.

First check that the v3dconverter stuff is correctly committed in the other branch.

Then you can do `git rebase -i master` on this branch and select which commits to drop. I can do it for you too.

Reply:
If you can do it, I think that it will be less dangerous for the SarII project... This morning I read some documentation about rebase, but frankly speaking, it is not clear for me.